src/DiceRoller/Dice.hpp

```diff
@@ -0,0 +1,63 @@
+//
+//  Dice.hpp
+//  DiceRoller
+//
+//  Created by Francesco Benzi on 2019-09-20.
+//  Copyright © 2019 Francesco Benzi. All rights reserved.
+//
+
+#ifndef Dice_hpp
+#define Dice_hpp
+
+#pragma once
+#include <vector>
+#include <stdio.h>
+
+class Dice
+{
+    
+public:
+    
+    Dice();
+    
+    /*
+     * diceRoller() rolls one to three dice, registers the new %
+     * and returns a sorted container which holds the
+     * values of the dice.
+     */
+    std::vector<int> diceRoller(int numDice);
```

Comment:
not a used thing, but I think roll(...) would be a better signature for that method...

```
Dice dice;
dice.roll(3);
```

sounds better than

```
Dice dice;
dice.diceRoller(3);
```

I know it's really subjective, so if anyone have a better idea let us know

Reply:
I agree with Kerry here...

Reply:
I agree as well, I renamed it to _dice()_.
I think I left it like that because it was the name of my local project.